app/views/payments/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<p style="color: green"><%= notice %></p>
+
+<h1 class='title-categories'>Payments</h1>
+<div class='new-payment'><%= link_to "New payment", new_payment_path %></div>
+
+<div id="payments">
+  <% @payments.each do |payment| %>
+    <%= render payment %>
+    <p>
+      <%= link_to "Show this payment", payment %>
+    </p>
+  <% end %>
+</div>
```

Comment:
- [ ] The payments page also does not follow the design and payments made do not appear accordingly.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/221279311-1d51601b-4477-474f-a0b8-3bb4831b1c1f.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/221275496-6492901f-52db-4e86-a5c6-f33135cd36c5.png)